### ISSUE TITLE
Double option: --skip --skip

### ISSUE THREAD

**Issue by DimitriPapadopoulos** [codespell-project/codespell#2060]
When passing two **`-S`** arguments to codespell, only the last one is taken into account. New patterns should be appended, not overwritten.

**Comment by DimitriPapadopoulos** [comment]
I cannot reproduce this with a simple case, I will keep investigating what happens in my complex case. Additionally, the [source code looks good at first sight](https://github.com/codespell-project/codespell/blob/f3901ed/codespell_lib/_codespell.py#L340-L341):
```python
    parser.add_argument('-S', '--skip',
                        action='append',
```

**Comment by DimitriPapadopoulos** [comment]
I cannot reproduce that anymore. I believe I must have been confused by issues described in #1915 and #2047, and hopefully fixed by pull request #2058.
